.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,110 @@
+name: Continuous Integration
+
+on: [push, pull_request]
+
+jobs:
+  unix-build:
+    name: Unix Build
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, macOS-latest]
+        python-version: [3.7, 3.8]
+    runs-on: ${{ matrix.os }}
```

Comment:
@michdolan for the copyright statement, if the work is under a different project's governance then your statement adjustment makes sense; otherwise I'd just keep it as ```OpenColorIO Project``` as before

Reply:
@michdolan : Yes, absolutely! I went through past of least resistance here, all the colour-science repositories are using the same CI code and I knew it would work out of the box. This is typically the type of tasks we could distribute easily to a persone.

Reply:
> @michdolan for the copyright statement, if the work is under a different project's governance then your statement adjustment makes sense; otherwise I'd just keep it as `OpenColorIO Project` as before

Whatever is simplest, it is merely a search'n'replace at this stage and I don't have a strong opinion here.